src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: JMX monitoring with ActiveMQ
+tags:
+  - Integrations
+  - On-host integrations
+  - On-host integrations list
+  - ActiveMQ dashboard
+  - ActiveMQ with New Relic
+metaDescription: 'Get a tour of our quickstarts and dashboards that take JMX-scraped data from your favorite automation software and houses it in the New Relic platform.'
+--- 
+
+import activemqLandingPage from 'images/activemqLandingPage.png'
+
+import accessDashboards from 'images/Access_dashboards.png' 
+
+With our ActiveMQ dashboard, you can easily track your ActiveMQ logs, keep an eye on your critical brokers, and get a total overview of all components of your messaging system. Built with our Linux infrastructure agent and our JMX integration, ActiveMQ dashboards take advantage of JMX scraping so you can view all your most important data, all in one place.
+
+<img
+  src={activemqLandingPage}
+  title="ActiveMQ dashboard landing page"
+  alt="A screenshot of a dashboard with ActiveMQ metrics. The dashboard ."
+/>
+<figcaption>
+After setting up ActiveMQ with New Relic, your data will display in a quickstart dashboard like this one, out of the box. 
+</figcaption> 
+
+## Set up the JMX integration [#setup]
+
+Before getting ActiveMQ data into New Relic, first install our infrastructure agent, then expose your metrics by installing the JMX integration. If you haven't already:
+
+* Follow our [guided install](https://one.newrelic.com/nr1-core?account=3501165&state=ff641d8b-e722-855e-4c4a-c50c23969dc3) to instrument your system with the Linux infrastructure agent. Or, if you prefer, [install the Linux infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) via the command line.
+* Install the [JMX monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-integration).
+
+## Get ActiveMQ metrics in a dashboard [#send]
+
+Our ActiveMQ dashboard takes data scraped from your JMX integration and translates it into a collection of premade graphs and charts. Housing all your data in the New Relic Platform gives you a full overview of your system's health and critical business points. 
+
+<img
+  title="Access_dashboards.png"
```

Comment:
I would add this below the 'install' instructions. i find it a little confusing being right above the install instructions. I also think it might be overkill; we tend to go minimal with screenshots, and if we took this approach we'd have similar screenshots in many many docs. I might just say 'After you're done installing, you can find your dashboards by going to the **Dashboards** tab in the top nav.' Something like that.